src/components/SocialBtns/SocialBtns.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import SocialBtns from './SocialBtns';
+
+describe('Component: SocialBtns', () => {
+  it('renders without crashing', () => {
+    const wrapper = shallow(<SocialBtns />);
```

Comment:
It could be useful to do a test or two to make sure the URLs you pass in as props appear in the rendered HTML.